bundles/CoreBundle/EventListener/SearchBackendListener.php

```diff
@@ -52,8 +54,10 @@ public static function getSubscribedEvents()
      */
     public function onPostAddElement(ElementEventInterface $e)
     {
-        $searchEntry = new Data($e->getElement());
-        $searchEntry->save();
+        $element = $e->getElement();
+        \Pimcore::getContainer()->get(MessageBusInterface::class)->dispatch(
```

Comment:
you are in an service here and still use Pimcore::getContainer(). Use DI instead.

Reply:
done